create_book.rb

```diff
@@ -0,0 +1,16 @@
+class CreateBook
+  def initialize(state)
+    @state = state
+    @book_list = @state[:book_list]
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
```

Comment:
- [ ] In keeping with the single-responsibility rule of SOLID, the sole responsibility of the **create_book** method should be to simply create books. The user inputs and inputs validations should be handled by another method. Kindly extract these input requests into another method.

  **SUGGESTION:** You can extract these requests methods into another method and have that method return and array of title and author. Then destructure the variables in create_book like so:

  ![anja-book](https://user-images.githubusercontent.com/36542986/157109135-5182a438-fc1e-40eb-99c2-e77a168d4779.jpg)